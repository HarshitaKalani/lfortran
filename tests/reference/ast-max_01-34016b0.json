{
    "basename": "ast-max_01-34016b0",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/errors/max_01.f90",
    "infile_hash": "96c2be2ccab6e4a1ed29b0d707ac35303f33d40c3f1fd2aace27dc62",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-max_01-34016b0.stdout",
    "stdout_hash": "0314874622931c33ce4c369c6b2d0cb1e349a9ebb74621e987fb29e5",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}