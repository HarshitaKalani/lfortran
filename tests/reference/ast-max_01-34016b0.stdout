(TranslationUnit
    [(Program
        max_01
        ()
        []
        []
        [(Declaration
            (AttrType
                TypeReal
                [(()
                8
                Value)]
                ()
                ()
                None
            )
            []
            [(y
            []
            []
            ()
            None
            ())]
            ()
        )
        (Declaration
            (AttrType
                TypeReal
                []
                ()
                ()
                None
            )
            []
            [(z
            []
            []
            ()
            None
            ())]
            ()
        )]
        [(=
            0
            y
            (Real "5.2d0")
            ()
        )
        (=
            0
            z
            (Real "9.0")
            ()
        )
        (Print
            0
            ()
            [(FuncCallOrArray
                max
                []
                [(()
                y
                ()
                0)
                (()
                z
                ()
                0)]
                []
                []
                []
            )]
            ()
        )]
        []
    )]
)
